qiime2/sdk/tests/test_method.py

```diff
@@ -396,9 +396,10 @@ def test_call_with_optional_artifacts(self):
 
         self.assertEqual(obs.view(list), [0, 42, 43, 42, 99, -22, 43, 43, 111])
 
+        # TODO: break this out into section with other "bad input" tests?
```

Comment:
Leftover dev comment?

Reply:
Yes. And related to the comment above (combining with `test_method.py`), is there value in digging through `test_method`, `test_pipeline`, etc to consolidate or remove type mismatch tests that are covered more systematically by the tests in this PR?

Reply:
I vote for leaving the existing actions alone.

Reply:
🗸